app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/sample/sample_status_events_table.scala.html

```diff
@@ -44,7 +44,16 @@ <h3 class="heading-medium" id="sample-status-events-heading">Sample activity</h3
                             val body = event.details.asInstanceOf[SampleStatusChange]
                             progressive_disclosure(
                                 id = s"sample-status-events-row-$index",
-                                summary = SampleStatus.format(body.to),
+                                summary = s"Sample status changed from ${SampleStatus.format(body.from, false)} to ${SampleStatus.format(body.to, false)}",
+                                body = body.comment.map(Html(_))
```

Comment:
It didn't support it at the time of you putting this in - but progressive disclosure can now support tabindexes, so we should probably set them here to allow for expanding these elements using keyboard nav.

Reply:
Good spot

Reply:
Fixed.   But the progressive bit doesn't appear to be used 'cos there is no 'comment'.  But done anyway